nnpdf_data/nnpdf_data/filter_utils/athena_utils.py

```diff
@@ -0,0 +1,131 @@
+from pathlib import Path
+from typing import Optional, Union
+
+import numpy as np
+import pandas as pd
+import yaml
+
+
+def read_excel(path_xlsx: Path, beams: tuple) -> pd.DataFrame:
```

Comment:
@Radonirinaunimi why is this stored in excel files? Since we're using git we should probably avoid that where possible.

Reply:
Unfortunately, that's the format in which these projections are provided. We can have another external script that transforms these into more trackable files and these are the ones used in the filters, but then I am not sure if we want that since the actual raw data and the external script will not be tracked (?)

Reply:
> Unfortunately, that's the format in which these projections are provided

Where are they taken from? I agree that translating the rawdata to another format before then passing it through the filter.py into yet another format is not the cleanest thing to do. Fortunately, them being rawdata files we will never touch them. Unfortunately, it means we require some program to read the .xlsx files.

Reply:
> Where are they taken from? I agree that translating the rawdata to another format before then passing it through the filter.py into yet another format is not the cleanest thing to do.

They were provided to us by the Athena collaboration during the FONLL POL studies.

> Fortunately, them being rawdata files we will never touch them. Unfortunately, it means we require some program to read the .xlsx files.
 
Indeed.